app/src/components/VennDiagram/VennDiagram.tsx

```diff
@@ -0,0 +1,76 @@
+import { stageFourEllipsisOn } from 'components/VennDiagram/venn/fourSets';
+import { stageThreeCirclesOn } from 'components/VennDiagram/venn/threeSets';
+import { VennDiagramTooltip } from 'components/VennDiagram/venn/tooltip';
+import { stageTwoCirclesOn } from 'components/VennDiagram/venn/twoSets';
+import { VennFourSetsPayload } from 'components/VennDiagram/venn/types/fourSetsTypes';
+import { VennThreeSetsPayload } from 'components/VennDiagram/venn/types/threeSetsTypes';
+import { VennTwoSetsPayload } from 'components/VennDiagram/venn/types/twoSetsTypes';
+import {
+  VennDiagramFlavors,
+  VennDiagramProps,
+} from 'components/VennDiagram/VennDiagramProps';
+import * as d3 from 'd3';
+import React, { useEffect } from 'react';
+
+export const VennDiagram = ({
+  flavor,
+  payload,
+}: VennDiagramProps): JSX.Element => {
+  // Attention - will be updated for prop update!
+  let svgElement: SVGSVGElement | null = null;
+  let tooltip: VennDiagramTooltip | undefined = undefined;
```

Comment:
Do not use let within a functional component

Reply:
uses useRef and useState now